clang/tools/clang-linker-wrapper/LinkerWrapperOpts.td

```diff
@@ -126,3 +126,8 @@ def version : Flag<["--", "-"], "version">, Flags<[HelpHidden]>, Alias<v>;
 
 def whole_archive : Flag<["--", "-"], "whole-archive">, Flags<[HelpHidden]>;
 def no_whole_archive : Flag<["--", "-"], "no-whole-archive">, Flags<[HelpHidden]>;
+
+// COFF-style linker options.
+def out : Joined<["/", "-", "/?", "-?"], "out:">, Flags<[HelpHidden]>;
```

Comment:
Why `/?` and `-?` in the permitted option prefix list? Wouldn't this allow e.g. `-?out:`?

Reply:
Feels strange to me, but justification is in 4b81e9f8d10214fbb2c8e1a367a60801a5dd5fed